packages/cfd/src/Stores/Modules/CFD/Helpers/cfd-config.ts

```diff
@@ -110,7 +110,18 @@ export const getMtCompanies = (is_eu: boolean) => {
             },
         },
         real: {
-            // TODO: Add All here before real release
+            all: {
```

Comment:
Why do we need "all" and "dxtrade" while they have the same values ?

Reply:
These two keys are used in two different places. This object is refactored in [this](https://github.com/binary-com/deriv-app/pull/6636) PR.